pwiz_tools/Skyline/TestData/Results/SmallWiffTest.cs

```diff
@@ -47,22 +47,67 @@ public void FileTypeTest()
         {
             var testFilesDir = new TestFilesDir(TestContext, ZIP_FILE);
 
-            string extWiff = ExtensionTestContext.ExtAbWiff;
-            string suffix = ExtensionTestContext.CanImportAbWiff ? "" : "-test";
-
-            // Do file type checks
-            using (var msData = new MsDataFileImpl(testFilesDir.GetTestPath("051309_digestion" + suffix + extWiff)))
+            // wiff1
             {
-                Assert.IsTrue(msData.IsABFile);
+                string extWiff = ExtensionTestContext.ExtAbWiff;
+                string suffix = ExtensionTestContext.CanImportAbWiff ? "" : "-test";

```

Comment:
All this suffix handling looks like a hassle. I usually just remove the sample-name suffix appended by MSConvert to mzML from WIFF files.

Reply:
Agreed, I was just following the wiff1 treatment already there. Is it worth adding another 4mb to the repo though? (FYI that would be one advantage I didn't previously consider if each developer did PRs from their own forks: intermediate updates of large files won't all be stored in the main repo; I think even if we squash on merging the intermediate commits stay in the repo, in limbo; getting rid of them requires a history rewrite)

Reply:
Okay